src/dcm/agent/plugins/builtin/unmount_volume.py

```diff
@@ -17,24 +17,43 @@
 import dcm.agent.plugins.api.base as plugin_base
 import dcm.agent.plugins.api.utils as plugin_utils
 import dcm.agent.utils as utils
+import logging
 
+_g_logger = logging.getLogger(__name__)
+
+_device_translate_map = {
+    "ubuntu": lambda x: 'xvdh' if x == 'sdh' else x,
```

Comment:
It may not be h.  It could be sdj or sdd or whatever.

I personally don't like inline lambda's like taht but it is fine if we can keep it that small and readable.